lib/src/room.dart

```diff
@@ -2090,27 +2090,38 @@ class Room {
   /// Returns the [PushRuleState] for this room, based on the m.push_rules stored in
   /// the account_data.
   PushRuleState get pushRuleState {
-    final globalPushRules =
-        client.accountData['m.push_rules']?.content['global'];
-    if (globalPushRules is! Map) {
+    final globalPushRules = client.globalPushRules;
+    if (globalPushRules == null) {
```

Comment:
Just for my understanding could you explain why when the globalpushrules is null we return the notify state?

Reply:
Added a comment :)